### PR TITLE
Enable netstandard

### PR DESCRIPTION
**Pull request by danielmarbach**
Small step towards #56 

Brings in netstandard but does not yet execute all the test for netcore as well but that is a good first step

### CODE REVIEW

File: src/TestRunner/Result.cs [hunk 1]
```diff
@@ -22,25 +22,18 @@ public Result(string output = null, Exception exception = null)
             HasException = exception != null;
         }
 
-        [DataMember]
-        public bool HasOutput { get; set; }
+        [DataMember] public bool HasOutput { get; set; }
```
Comment (timbussmann) [inline src/TestRunner/Result.cs]: that is kinda weird formatting?

File: src/TestRunner/TestRunner.csproj [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net452</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (timbussmann) [inline src/TestRunner/TestRunner.csproj]: I have to admit that I'm not familiar with the project structure of the repo, but seing a "runner" project targeting netstandard confuses me a bit.

Comment (danielmarbach) [reply src/TestRunner/TestRunner.csproj]: It is a lib for the base runner

File: src/TestRunner/Result.cs (hunk 1 above)
Comment (danielmarbach) [reply src/TestRunner/Result.cs]: Happy to reset that file

**Comment by danielmarbach** [comment]
There is basically no risk if we merge this as is. We don't multi target at all and we would only be testing the MS SDK essentially under netcore. I'm happy to look at the deployment that needs to change in order to support netcore as a dedicated PR. But even if we don't have it this PR provides value.

**Comment by danielmarbach** [comment]
Did some more research and SDK project style is still not supported, see https://github.com/Azure/service-fabric-issues/issues/558

We continue to only execute the tests on full framework for now but at least we have netstandard support

**Pull request merged**